packages/EasyStandard/src/Sniffs/ControlStructures/LinebreakAfterEqualsSignSniff.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Sniffs\ControlStructures;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+final class LinebreakAfterEqualsSignSniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    private const LINEBREAK_AFTER_EQUALS_SIGN = 'LinebreakAfterEqualsSign';
+
+    /**
+     * @param int $equalsSignPointer
+     */
+    public function process(File $phpcsFile, $equalsSignPointer): void
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        $afterEqualsSignToken = $tokens[$equalsSignPointer + 1];
+
+        if ($afterEqualsSignToken['content'] !== "\n") {
+            return;
+        }
+
+        $phpcsFile->addErrorOnLine(
+            'The line can\'t be broken just after the equals sign.',
```

Comment:
Please change to `The line must not be broken right after the equals sign.`

Reply:
done